src/createServer.js

```diff
@@ -1,11 +1,252 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
 
 function createServer() {
   // Use express to create a server
   // Add a routes to the server
   // Return the server (express app)
+  const app = express();
+
+  let users = [];
+  // let users = [
+  //   {
+  //     id: 1,
+  //     name: 'Misha',
+  //   },
+  //   {
+  //     id: 2,
+  //     name: 'Dasha',
+  //   },
+  //   {
+  //     id: 3,
+  //     name: 'Lesya',
+  //   },
+  // ];
+
+  app.get('/users', express.json(), async (req, res) => {
```

Comment:
Using `express.json()` middleware in a GET request is unnecessary. This middleware is used to parse JSON bodies in requests, which is not needed for GET requests.